core/class.go

```diff
@@ -61,7 +62,24 @@ type Cairo1Class struct {
 	Program         []*felt.Felt
 	ProgramHash     *felt.Felt
 	SemanticVersion string
-	Compiled        json.RawMessage
+	Compiled        CompiledClass
+}
+
+type CompiledClass struct {
+	Bytecode        []*felt.Felt
+	PythonicHints   json.RawMessage
```

Comment:
I think it is important to signify to the reader that the value stored there is not some arbitrary byte array but valid JSON.

Reply:
> that the value stored there is not some arbitrary byte array but valid JSON.

I agree with Omer here, IMO right now we do not rely on a lot of external types in `core` and since `json.RawMessage` is just stdlib type over `[]byte` it has no overhead and shouldn't cause much trouble (like cyclic dependency)